tests/testsuite

```diff
@@ -63,6 +63,28 @@ can_load_relative_paths()
         fail "failed to pass template by relative path"
 }
 
+respects_umask()
+{
+
+    local wd="$testdir/SAMPLE"
+    local orig_umask=`umask`
+    echo "testing: dg respects umask"
+    for i in 0000,-rw-rw-rw- 0002,-rw-rw-r--; do
+        IFS="," read mask bits <<< $i
+        umask $mask
+        local cmd="dg --projectdir '$wd' \
+           --distro 'rhel-7-x86_64.yaml' \
+           --spec '$wd/test.yaml' \
+           --template '$wd/test.tpl' \
+           --output '$wd/.output'"
+
+        eval "$cmd"
+	perm_bits=`stat -c "%A" $wd/.output`
+	[ "$perm_bits" == "$bits" ] || \
```

Comment:
Nit: here are tabs, not spaces

Since we don't pay attention to shell compatibility (we have `/bin/bash` shebang already), you can use a bit safer variant:

```bash
cmd=( dg --projectdir "$wd" ...)
"${cmd[@]}"
```